stile/sys_tests.py

```diff
@@ -692,6 +705,160 @@ def __call__(self, data, data2=None, random=None, random2=None, config=None, **k
         return self.getCF('gg', new_data, new_data2, new_random, new_random2,
                           config=config, **kwargs)
 
+class Rho2SysTest(BaseCorrelationFunctionSysTest):
+    """
+    Compute the correlation of star shapes with residual star shapes (star shapes - psf shapes).
+    """
+    short_name = 'rho2'
+    long_name = 'Rho2 statistics (Auto-correlation of star-PSF shapes)'
+    objects_list = ['star PSF']
+    required_quantities = [('ra', 'dec', 'g1', 'g2', 'psf_g1', 'psf_g2', 'w')]
+
+    def __call__(self, data, data2=None, random=None, random2=None, config=None, **kwargs):
```

Comment:
rho2 is correlation between PSF shapes and star-PSF shapes, I think we should put `data['psf_g1']` to `new_data['g1']` and then put `new_data` to `self.getCF()`.